cmake/download_viame_addons.csv

```diff
@@ -6,3 +6,4 @@ EM Tuna,https://viame.kitware.com/api/v1/item/627b326cc4da86e2cd3abb5b/download,
 MOUSS Deep 7,https://viame.kitware.com/api/v1/item/627b3282c4da86e2cd3abb5d/download,MOUSS deep7 fish detection in hawaii
 Aerial Penguin,https://viame.kitware.com/api/v1/item/615bc7aa7e5c13a5bb9af7a7/download,Aerial penguin detection from fixed wings/drones
 Sea Lion,https://viame.kitware.com/api/v1/item/641d644abfe642e7e4f54655/download,Sea lion aerial detection data
+PoolCar,https://viame.kitware.com/api/v1/file/646b5abb4c19222431def881/download, 10 shot Poolcar detector
```

Comment:
The poolcar package here isn't in the same format as the other add-ons, which would be putting any models in a 'config/pipelines/models' folder (this is so that it can be used as an add-on by users downloading it from the installation page, as opposed to just being used in the build system) - I'd probably reccomend changing the package to this format so we can also put them on the install page and here, considering they're 4 Gb. The format that they're in right now would be fine if their a default model in the installer but they won't be due to size.

Reply:
I reuploaded the model with that format, so it should be in the right format now